capstone-frontend/src/components/AssignmentCard.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { Button, Col, Form, Row } from 'react-bootstrap';
-
+import moment from 'moment';
```

Comment:
What is the moment import?

Reply:
This is the npm package that formats dates.